extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -380,17 +395,20 @@ public function categoryTopArticles() {
 		$pages = [];
 		$category = '';
 		$result = $this->getVal( 'result' );
-		if ( ! empty( $result ) ) {
+		if ( !empty( $result ) ) {
 			try {
 				$category = $result['title'];
 				$colonSploded = explode( ':', $category );
-				$namespace = (new Wikia\Search\MediaWikiService)->getNamespaceIdForString( $colonSploded[0] );
+				$namespace = ( new Wikia\Search\MediaWikiService )->getNamespaceIdForString( $colonSploded[0] );
 				// remove "Category:", since it doesn't work with ArticlesApiController
-				$category = ( is_int( $namespace ) && $namespace == NS_CATEGORY )
-						 ? implode( ':', array_slice( $colonSploded, 1 ) )
-						 : $category;
+				$category = ( is_int( $namespace ) && $namespace == NS_CATEGORY ) ?
+					implode( ':', array_slice( $colonSploded, 1 ) ) : $category;
 				//@todo use single API call here when expansion is released
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)